source/opt/pass_manager.cpp

```diff
@@ -62,7 +67,6 @@ Pass::Status PassManager::Run(opt::IRContext* context) {
   if (status == Pass::Status::SuccessWithChange) {
     context->module()->SetIdBound(context->module()->ComputeIdBound());
   }
-  passes_.clear();
```

Comment:
This is a second, possibly, visible change (and was causing a linker test error). Previously the pass manager would dump all pass information when it was done so you could re-use the manager by setting up everything again.

This CL leaves the Tokens in place so the manager can be re-used with a given set of passes.

I can change this back to clearing if preferred, we just need to decide if we want the manager to b re-usable with new passes, or with the previously configured passes.